docs/HTML.md

```diff
@@ -0,0 +1,93 @@
+### 1. `<header>` -- Header 
+* *type:* block-level
+* *content:* Flow content/palpable content
+* *support:* all browsers
+* *example:* 
+```
+ <header> Logo </header>
+```
+<header> Logo </header>
+
+### 2. `<hr>` -- Horizontal Rule
+* *type:* block-level
+* *content:* Flow Content
+* *support:* all browsers
+* *example:*
+``` <p> Some text </p>
+ <hr>
+<p> some other text </p> ```
+<p> some Text </p>
+<hr>
+<p> some other text </p>
+
+### 3. `<div>` -- Document Division
+* *type:* block-level
+* *content:* Flow content/palpable content
+* *support:*  all browsers
+* *example:*
+```<div> <p> Div Example </p> </div> ```
+<div> </p> Div Example </p> </div>
+
+### 4. `<address>` -- Address Element
+* *type:* block-level
+* *content:* flow content/palpable cotent
+* *support:* all browsers
+* *example:*
+```<address> 5555 Hugh Gerection Street <br> Somewhere, FL 55555 </adress>```
+<address> 5555 Hugh Gerection Street <br> Somewhere, FL 55555</address>
+
+### 5. `<ul>` -- Unordered List
+* *type:* block-level
+* *content:* flow content
+* *support:* all browsers
+* *example:*
+```<ul> <li> Thing One </li> <li> Thing Two </li> <li> Thing Three </li> </ul>```
+<ul> <li> Thing One </li> <li> Thing Two </li> <li> Thing Three </li> </ul>
+
+### 6. `<ol>` -- Ordered List
+* *type:* block-level
+* *content:*  flow content/palpable content
+* *support:*  all browsers
+* *example:*
+```<ol> <li> One Thing </li> <li> Two Thing </li> <li> Three Thing </li> </ol>```
+<ol> <li> One Thing </li> <li> Two Thing </li> <li> Three Thing </li> </ol>
+
+### 6. `<blockquote>` -- Long Block Quotation 
+* *type:* block-level
+* *content:* flow content
+* *support:* all browsers
+* *example:*
+ ```<blockquote> <p>Holler at your boy</p> </blockquote>```
+<blockquote><p> Holler at your boy </p> <blockquote>
+
+### 7. `<article>` -- Article Content
+* *type:* block-level
+* *content:* flow content/palpable content/sectioning content
+* *support:* all browsers
+* *example:*
+```<article> <h4> Heading </h4> <p> Articles should always include a heading </p> </article>```
+<article> <h4> Heading </h4> <p> Articles should always include a heading </p> </article>
+
+### 8. `<dl>` -- Definition List
+* *type:* block-level 
+* *content:*  flow content/palpable content
+* *support:* all browsers
+* *example:*
+```<dl> <dt> Challenge </dt> <dd> A call to take part in contest or competition </dd> </dl> ```
+<dl><dt>Challenge</dt><dd>A call to take part in contest or competition </dd></dl>
+
```

Comment:
Nice to see you start with the `<dl>` tag... Many others did not. Like above, this would be a good place to document the related tags and share an example block between them, particularly since a `<dl>` by itself is pretty worthless.